src/Element/NodeElement.php

```diff
@@ -305,37 +305,52 @@ public function blur()
         $this->getDriver()->blur($this->getXpath());
     }
 
+    /**
+     * Press one or more keyboard keys on an element.
+     *
+     * @param string|int|array $keys a string to type ('PHP Hypertext Preprocessor'), a integer key code (88), or an array of keys to press in sequence (e.g.: ["\u{E008}", 'H', "\u{E001}", 'ello']). See https://w3c.github.io/webdriver/#keyboard-actions
+     */
+    public function pressKey($keys) {
+        $this->getDriver()->pressKey($this->getXpath(), $keys);
+    }
+
     /**
      * Presses specific keyboard key.
      *
      * @param string|int  $char     could be either char ('b') or char-code (98)
      * @param string|null $modifier keyboard modifier (could be 'ctrl', 'alt', 'shift' or 'meta')
+     *
+     * @deprecated use pressKey($keys) instead.
      */
     public function keyPress($char, $modifier = null)
     {
-        $this->getDriver()->keyPress($this->getXpath(), $char, $modifier);
+        $this->pressKey($this->wrapCharWithModifier($char, $modifier));
```

Comment:
Adding a `@deprecated` tag alone in the DocBlock won't help much, but having a matching deprecation notice triggering code will.

---

The same goes for other similar places in this class.